shared/profile/edit-profile/index.js

```diff
@@ -35,6 +35,7 @@ export default compose(
     bioLengthLeft: props.bio ? maxProfileBioChars - props.bio.length : maxProfileBioChars,
   })),
   withHandlers({
+    onCancel: ({onBack}) => () => onBack(),
```

Comment:
you likely want to actually use https://github.com/acdlite/recompose/blob/master/docs/API.md#renameprop